guide/12-enrich-data-with-thematic-information/part6_country_enrich.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
```

Comment:
<p>As in the example from the first Notebook, <strong>study areas</strong> can be polygons defined manually and provided as input. They can also be standard geographic areas defined with the unique identifiers for the areas, such as postal (ZIP) codes. Finally, as is the case with this example, study areas can be provided as lines or points. Since lines and points do not define an area, we need to create polygons on the server to use for apportioning data to each location. </p><p>The polygons created around the lines and points is a five kilometer straight-line buffered area by default. This can be controlled using the <code>proximity</code> parameters of the enrich method: <code>proximity_type</code>, <code>proximity_value</code>, <code>proximity_metric</code>. For line geometries, only the straight line method can be used, while for point geometries, any transportation network method available in the GIS can be used to define the area surrounding the points. </p><h4><br></h4><h4>Discover Available Travel Modes </h4><p> </p><p>In this example, we know customers are willing to travel about eight minutes to visit our store. The enrich method is capable of creating eight-minute drive time areas around the potential store locations, but we need to know how to provide the correct inputs. For this example, we can discover the available travel modes using the <code>travel_modes</code> property.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>